internal/app/singularity/remote_status.go

```diff
@@ -31,7 +35,7 @@ type status struct {
 // RemoteStatus checks status of services related to an endpoint
 // If the supplied remote name is an empty string, it will attempt
 // to use the default remote.
-func RemoteStatus(usrConfigFile, name string) (err error) {
+func RemoteStatus(usrConfigFile, name string, libraryClientConfig *scslibclient.Config) (err error) {
```

Comment:
I could be wrong here, as I haven't tested this yet, but I don't think you need to set a `scslibclient.Config` here, because the following block is likely about to constuct the URL from remote setting.

```
var e *endpoint.Config
	if name == "" {
		e, err = c.GetDefault()
	} else {
		e, err = c.GetRemote(name)
	}
```

I suspect you can just add the `printLoggedInIdentity` to the `doTokenCheck` method and pull the lib URI from `e`.

Again, I might be incorrect.

Reply:
Ya, this LGTM.